include/dblayer/mysqli_innodb.php

```diff
@@ -26,10 +26,10 @@ class DBLayer
 
 	var $datatype_transformations = array(
 		'%^SERIAL$%'	=>	'INT(10) UNSIGNED AUTO_INCREMENT'
-	);
-
-
-	function DBLayer($db_host, $db_username, $db_password, $db_name, $db_prefix, $p_connect)
+	)
+	
+	
+	function __construct($db_host, $db_username, $db_password, $db_name, $db_prefix, $p_connect)
```

Comment:
This doesn't look right. You're removing a semicolon (also adding unneeded whitespace).

Plus, in most cases, you're removing an empty line above the new constructors (there should be two blank lines for consistency).

Thanks again!